cmd/create/oidcprovider/cmd.go

```diff
@@ -203,15 +204,20 @@ func buildCommands(r *rosa.Runtime, cluster *cmv1.Cluster) (string, error) {
 	}
 	r.Reporter.Debugf("Using thumbprint '%s'", thumbprint)
 
-	tag := fmt.Sprintf(
-		"Key=%s,Value=%s",
-		tags.ClusterID, cluster.ID())
-	createOpenIDConnectProvider := fmt.Sprintf("aws iam create-open-id-connect-provider \\\n"+
-		"\t--url %s \\\n"+
-		"\t--client-id-list %s %s \\\n"+
-		"\t--thumbprint-list %s \\\n"+
-		"\t --tag %s",
-		oidcEndpointURL, aws.OIDCClientIDOpenShift, aws.OIDCClientIDSTSAWS, thumbprint, tag)
+	tag := map[string]string{
+		tags.ClusterID: cluster.ID(),
+	}
+
+	clientIdList := strings.Join([]string{aws.OIDCClientIDOpenShift, aws.OIDCClientIDSTSAWS}, " ")
+	clientIdList = clientIdList[0 : len(clientIdList)-1]
```

Comment:
This isn't needed, `strings.join` won't add a space at the end, you'll be dropping the last char of `aws.OIDCClientIDSTSAWS`.